File: JavaScript Fundamentals/Homeworks/Conditional Statements/05. Digit As Word/digitAsWord.js
function digitAsWord() {
    'use strict';

    var input = +document.getElementById('inputForm').value;

    switch (input) {

        case 0: window.console.log('Zero');
            document.getElementById('console').innerHTML += 'Zero' + '<br>';
            break;
        case 1:
            window.console.log('One');
            document.getElementById('console').innerHTML += 'One' + '<br>';
            break;
        case 2:
            window.console.log('Two');
            document.getElementById('console').innerHTML += 'Two' + '<br>';
            break;
        case 3:
            window.console.log('Three');
            document.getElementById('console').innerHTML += 'Three' + '<br>';
            break;
        case 4:
            window.console.log('Four');
            document.getElementById('console').innerHTML += 'Four' + '<br>';
            break;
        case 5:
            window.console.log('Five');
            document.getElementById('console').innerHTML += 'Five' + '<br>';
            break;
        case 6:
            window.console.log('Six');
            document.getElementById('console').innerHTML += 'Six' + '<br>';
            break;
        case 7:
            window.console.log('Seven');
            document.getElementById('console').innerHTML += 'Seven' + '<br>';
            break;
        case 8:
            window.console.log('Eight');
            document.getElementById('console').innerHTML += 'Eight' + '<br>';
            break;
        case 9:
            window.console.log('Nine');
            document.getElementById('console').innerHTML += 'Nine' + '<br>';
            break;
        default:
            window.console.log('Not a digit');
            document.getElementById('console').innerHTML += 'Not a digit' + '<br>';
    }
}

File: CSharp Part I/05. Conditional Statements/08. Digit As Word/DigitAsWord.cs
// Write a program that read a digit (0-9) from the console, and depending on the input, shows the digit as a word (in English).

namespace _08.Digit_As_Word
{
    using System;

    public class DigitAsWord
    {
        static void Main()
        {
            string number = Console.ReadLine();

            switch (number)
            {
                case "1": Console.WriteLine("one"); break;
                case "2": Console.WriteLine("two"); break;
                case "3": Console.WriteLine("three"); break;
                case "4": Console.WriteLine("four"); break;
                case "5": Console.WriteLine("five"); break;
                case "6": Console.WriteLine("six"); break;
                case "7": Console.WriteLine("seven"); break;
                case "8": Console.WriteLine("eight"); break;
                case "9": Console.WriteLine("nine"); break;
                case "0": Console.WriteLine("zero"); break;
                default: Console.WriteLine("not a digit"); break;
            }
        }
    }
}
